src/App.js

```diff
@@ -40,6 +61,13 @@ class App extends Component {
                                 <h1>Hi</h1>
                             </div>
                             <ScrollableView>
+                                <div className={addPlayerForm}>
+                                    {
```

Comment:
I would personally wrap all of this inside its own component so that it can manage the state of switching between the button and input. It's good to have separation of concerns, App should deal with managing the playerList only, so keep the methods for addingPlayers but take the boolean into its own component.